.idea/todo_api.iml

```diff
@@ -47,6 +47,7 @@
     <orderEntry type="library" scope="PROVIDED" name="debug (v1.5.0, rbenv: 3.1.2) [gem]" level="application" />
```

Comment:
https://git-scm.com/docs/gitignore

Reply:
I removed the .idea files and added it to .gitignore in these 2 commits from branch 10-add-rubocop already:
775e30ea064c6599023c577b259cf7b87180e852
505a01b432945df1e20909a642b09fb19db1a4ca